### PR TITLE
add the Agilex LIMO into support

### PR DESCRIPTION
**Pull request by agilexrobotics**

### CODE REVIEW

File: _posts/2021-12-10-limo.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: LIMO
+date: 2021-12-10
+description:
+icon: /assets/img/robots/limo/limo.jpg
```
Comment (tfoote) [inline _posts/2021-12-10-limo.md:6]: Please make the icon 80x80 pixels. You can embed the additional picture in the main body. See the comment on the TODO lower.

File: _posts/2021-12-10-limo.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+- 'platform'
+- 'raspberry pi'
+- 'robotis'
+- 'teaching robot'
+- 'tracking'
+twitter_text: Checkout the LIMO
+website: https://global.agilex.ai/products/limo
+wiki_homepage: https://github.com/agilexrobotics/limo-doc
+---
+TODO Fill in the long description/article here.
```
Comment (tfoote) [inline _posts/2021-12-10-limo.md]: The content for this page is just at TODO. It needs more information. I'd recommend some descriptions possibly some images and some video embeds for the best engagement.

**Review by tfoote — CHANGES_REQUESTED** [review]
Please take a moment to fill in the main body and fix the icon size. The icon needs to be small to be able to fit onto the card. 

There's some guidelines to help guide contributions here: https://robots.ros.org/contributing/

**Review by agilexrobotics — COMMENTED** [review]
update image

File: _posts/2021-12-10-limo.md (hunk 1 above)
Comment (agilexrobotics) [reply _posts/2021-12-10-limo.md:6]: the new image i have push to the laster version

File: _posts/2021-12-10-limo.md (hunk 2 above)
Comment (agilexrobotics) [reply _posts/2021-12-10-limo.md]: New veison i have add the description into the md file

**Review by tfoote — APPROVED** [review]
Thanks for the updates.

**Pull request merged**